### PR TITLE
[ODS-4299] Cleanup remaining bulk references

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/EdFi.Ods.WebService.Tests/App.config [hunk 1]
```diff
@@ -20,10 +20,7 @@
     <!-- Uncomment WebDriver setting to see UI tests run in Firefox-->
     <!--<add key="WebDriver" value="Firefox"/>-->
     <!-- Sql Server 2008 by Default-->
-    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />
-    <add key="QueuePrefix" value="EdFiOds" />
-    <add key="QueueAutoCreate" value="1" />
-    <add key="CommitUploadCommandMessageEndPoint" value="localhost" />
+    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />
```
Comment (blmeyers) [inline Application/EdFi.Ods.WebService.Tests/App.config:23]: This one is also for bulk

File: Application/EdFi.Ods.WebService.Tests/App.config [hunk 2]
```diff
@@ -20,10 +20,7 @@
     <!-- Uncomment WebDriver setting to see UI tests run in Firefox-->
     <!--<add key="WebDriver" value="Firefox"/>-->
     <!-- Sql Server 2008 by Default-->
```
Comment (blmeyers) [inline Application/EdFi.Ods.WebService.Tests/App.config:22]: Also for bulk

File: Application/EdFi.Ods.WebService.Tests/App.config [hunk 3]
```diff
@@ -20,10 +20,7 @@
...
-    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />
-    <add key="QueuePrefix" value="EdFiOds" />
-    <add key="QueueAutoCreate" value="1" />
-    <add key="CommitUploadCommandMessageEndPoint" value="localhost" />
+    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />        
     <add key="StartOperationCommandMessageEndpoint" value="localhost" />
```
Comment (blmeyers) [inline Application/EdFi.Ods.WebService.Tests/App.config:24]: Also for bulk

File: Application/EdFi.Ods.WebService.Tests/App.config [hunk 4]
```diff
@@ -20,10 +20,7 @@
...
     <add key="PreserveAllTestSdks" value="true" />
     <add key="ReleaseBulkDatabaseAfterProcessing" value="true" />
```
Comment (blmeyers) [inline Application/EdFi.Ods.WebService.Tests/App.config:26]: Also for bulk

File: Application/EdFi.Ods.WebService.Tests/App.config (hunk 1 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebService.Tests/App.config]: yes removed "SqlServerDataDirectory"

File: Application/EdFi.Ods.WebService.Tests/App.config (hunk 2 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebService.Tests/App.config]: yes removed 3 lines

File: Application/EdFi.Ods.WebService.Tests/App.config (hunk 3 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebService.Tests/App.config]: yes removed "StartOperationCommandMessageEndpoint"

File: Application/EdFi.Ods.WebService.Tests/App.config (hunk 4 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.WebService.Tests/App.config]: yes removed "ReleaseBulkDatabaseAfterProcessing"

**Comment by semalaiappan** [comment]
I removed .gitignore file: #File System Queues section
I removed Application\EdFi.Ods.WebService.Tests\WebApiInstaller.cs: line 99, commented out queue installer

**Pull request merged**